beacon-chain/slasher/rpc_test.go

```diff
@@ -125,7 +126,7 @@ func TestIsSlashableAttestation(t *testing.T) {
 		{
 			name:         "should detect multiple surround if multiple same indices",
 			attToCheck:   createAttestationWrapper(t, 1, 4, []uint64{0, 1}, []byte{2}),
-			amtSlashable: 2,
+			amtSlashable: 4,
```

Comment:
How can this be 4 if only two indices attested?

Reply:
4 slashable attestations were found. Two indices, each one has a surrounding and a surrounded attestation

Reply:
It's kind of weird, but yes slasher is technically finding 4 total attestations here